controllers/deadmanssnitchintegration/deadmanssnitchintegration_controller.go

```diff
@@ -330,21 +330,17 @@ func (r *DeadmansSnitchIntegrationReconciler) createSnitch(dmsi *deadmanssnitchv
 				return err
 			}
 
-			var snitch dmsclient.Snitch
 			if len(snitches) <= 0 {
 				newSnitch := dmsclient.NewSnitch(snitchName, dmsi.Spec.Tags, "15_minute", "basic")
 				newSnitch.Notes = fmt.Sprintf("cluster_id: %s\nrunbook: https://github.com/openshift/ops-sop/blob/master/v4/alerts/cluster_has_gone_missing.md", clusterID)
 				// add escaping since _ is not being recognized otherwise.
 				newSnitch.Notes = "```" + newSnitch.Notes + "```"
 				logger.Info(fmt.Sprint("Creating snitch:", snitchName))
-				snitch, err = dmsc.Create(newSnitch)
+				_, err = dmsc.Create(newSnitch)
```

Comment:
tbh, the whole case is in `k8errors.IsNotFound(err)`,  so you think it could be an issue where the k8s client got a momentary err for all of those existing snitches?

Reply:
I am not convinced this is the error even though it might. Nevertheless, the code as it is doesn't make sense (checking a snitch status to validate a different one potentially) so this deserves to be fixed in any case. 
Without logs or anything, it is difficult to investigate much more, and we didn't have reoccurrence (except maybe with activation/deactivation of CAD handling from PD stats) which rules out behaviour related to promotion/restart and prevent from digging further on recent case to get some logs.